src/main/java/com/denizenscript/ddiscordbot/commands/DiscordInteractionCommand.java

```diff
@@ -84,141 +87,104 @@ public DiscordInteractionCommand() {
 
     public enum DiscordInteractionInstruction { DEFER, REPLY, EDIT, DELETE }
 
-    @Override
-    public void parseArgs(ScriptEntry scriptEntry) throws InvalidArgumentsException {
-        for (Argument arg : scriptEntry) {
-            if (!scriptEntry.hasObject("instruction")
-                    && arg.matchesEnum(DiscordInteractionInstruction.class)) {
-                scriptEntry.addObject("instruction", arg.asElement());
-            }
-            else if (!scriptEntry.hasObject("message")) {
-                scriptEntry.addObject("message", arg.getRawObject());
-            }
-            else {
-                arg.reportUnhandled();
-            }
-        }
-    }
-
-    @Override
-    public void execute(ScriptEntry scriptEntry) {
-        ElementTag instruction = scriptEntry.getElement("instruction");
-        DiscordInteractionTag interaction = scriptEntry.requiredArgForPrefix("interaction", DiscordInteractionTag.class);
-        boolean ephemeral = scriptEntry.argAsBoolean("ephemeral");
-        ElementTag attachFileName = scriptEntry.argForPrefixAsElement("attach_file_name", null);
-        ElementTag attachFileText = scriptEntry.argForPrefixAsElement("attach_file_text", null);
-        ObjectTag rows = scriptEntry.argForPrefix("rows", ObjectTag.class, true);
-        ObjectTag message = scriptEntry.getObjectTag("message");
-        if (scriptEntry.dbCallShouldDebug()) {
-            // Note: attachFileText intentionally at end
```

Comment:
`// Note: attachFileText intentionally at end` this comment needs to be reinstated